### PR TITLE
P17 profiler moorings

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP01CNPM/CP01CNPM_D00005_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP01CNPM/D00005/imm/mmp/E*.DAT,CP01CNPM-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP01CNPM/D00005/imm/mmp/A*.DEC,CP01CNPM-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP01CNPM/D00005/imm/mmp/C*.DAT,CP01CNPM-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP01CNPM/D00005/imm/mmp/C*.DAT,CP01CNPM-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP01CNPM/D00005/imm/mmp/E*.DAT,CP01CNPM-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP01CNPM/D00005/imm/mmp/E*.DAT,CP01CNPM-WFP01-05-PARADK000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP01CNPM/CP01CNPM_D00005_ingest.csv:10]: looks good. Data available

File: CP02PMCO/CP02PMCO_D00016_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00016/imm/mmp/E*.DAT,CP02PMCO-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00016/imm/mmp/A*.DEC,CP02PMCO-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00016/imm/mmp/C*.DAT,CP02PMCO-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00016/imm/mmp/C*.DAT,CP02PMCO-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00016/imm/mmp/E*.DAT,CP02PMCO-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00016/imm/mmp/E*.DAT,CP02PMCO-WFP01-05-PARADK000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP02PMCO/CP02PMCO_D00016_ingest.csv:10]: looks good. Data available.

File: CP02PMUO/CP02PMUO_D00018_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00018/imm/mmp/E*.DAT,CP02PMUO-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00018/imm/mmp/A*.DEC,CP02PMUO-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00018/imm/mmp/C*.DAT,CP02PMUO-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00018/imm/mmp/C*.DAT,CP02PMUO-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00018/imm/mmp/E*.DAT,CP02PMUO-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMUO/D00018/imm/mmp/E*.DAT,CP02PMUO-WFP01-05-PARADK000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP02PMUO/CP02PMUO_D00018_ingest.csv:10]: Looks good. Data available.

File: CP03ISPM/CP03ISPM_D00005_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP03ISPM/D00005/imm/mmp/E*.DAT,CP03ISPM-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP03ISPM/D00005/imm/mmp/A*.DEC,CP03ISPM-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP03ISPM/D00005/imm/mmp/C*.DAT,CP03ISPM-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP03ISPM/D00005/imm/mmp/C*.DAT,CP03ISPM-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP03ISPM/D00005/imm/mmp/E*.DAT,CP03ISPM-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP03ISPM/D00005/imm/mmp/E*.DAT,CP03ISPM-WFP01-05-PARADK000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP03ISPM/CP03ISPM_D00005_ingest.csv:10]: Looks good. Data available.

File: CP04OSPM/CP04OSPM_D00015_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00015/imm/mmp/E*.DAT,CP04OSPM-WFP01-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00015/imm/mmp/A*.DEC,CP04OSPM-WFP01-01-VEL3DK000,telemetered,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00015/imm/mmp/C*.DAT,CP04OSPM-WFP01-02-DOFSTK000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00015/imm/mmp/C*.DAT,CP04OSPM-WFP01-03-CTDPFK000,telemetered,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00015/imm/mmp/E*.DAT,CP04OSPM-WFP01-04-FLORTK000,telemetered,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP04OSPM/D00015/imm/mmp/E*.DAT,CP04OSPM-WFP01-05-PARADK000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP04OSPM/CP04OSPM_D00015_ingest.csv:10]: Looks good.  Data available.

**Review by rtravis-whoi — APPROVED** [review]
Looks good. Data available.

File: CP03ISPM/CP03ISPM_D00005_ingest.csv (hunk 4 above)
Comment (snwhite) [reply CP03ISPM/CP03ISPM_D00005_ingest.csv:10]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**